envoy/api/v2/README.md

```diff
@@ -1 +1,9 @@
 Protocol buffer definitions for core API messages.
```

Comment:
Yeah, I think we need to rename this use of "core" as well.

Reply:
It's basically a placeholder for old legacy stuff we can't move and acts just as a parent node otherwise.

Reply:
made it clear this is just old xDS + top-level resources